lib/vagrant-service-manager/command.rb

```diff
@@ -34,7 +34,7 @@ def execute
           when 'docker'
```

Comment:
Can you combine the docker and openshift subcommands with a temporary extra check around --script-readable for openshift ... it'd be nice to get a follow up PR that removes that.